### DISCUSSION TITLE
win10 sync stops in full node

### DISCUSSION THREAD

**Discussion by tianxinya2021** [Chia-Network/chia-blockchain#2792] [General]
This morning I found two of my full nodes in win10 sync stopped and farm stopped, one since 9pm last night and the other since 2am this morning, but their plotting is still working. At the same time, two other full nodes are running fun. All four full nodes are on the same wallet, but each has its own internet ip. I tried to click 'force reload', but did not work. I had to quit both and restarted full node, but my pending plots are lost from gui. Is there anyway we can push re-sync without shutting down the full node? Thanks.

**Comment by cccat6** [comment]
Did you upgrade to 1.1.x?

**Reply by tianxinya2021** [reply]
The two with sync stopped had 1.1.1 and 1.0.5. Even after restart, the 1.1.1 did not initiate sync after 30 min. I had to re-run 1.0.5, which could sync about 30 min after re-start.

**Comment by dorkmo** [comment]
you should really only have one full node running per network

**Reply by tianxinya2021** [reply]
I am running them mostly to get plots, but at the same time, farming too. Each has its own internet ip. I do not see the reason why not. I do see a lot of discussion on multiple farm/plot PCs behind one private network to use only one full node. Appreciate any input on this.

**Reply by dorkmo** [reply]
oh if each are on their own internet ip then its fine. you probably want to do port 8444 forwarding if you have not already. how many peer nodes are you connected to on each of your nodes?

**Reply by tianxinya2021** [reply]
I thought the full node would connect to the chia net automatically. I do not understand the reason to connect to peers. Is there any need/advantage to connect to other unknown peers? what is port 8444 forwarding? Thanks.

**Reply by dorkmo** [reply]
Nodes sometimes can connect automatically to other peers, but lately so many new nodes are connecting it is pushing a lot of people offline.
the other unknown peers communicate constantly telling each other who has won a plot and other information. every 10 seconds a signal is sent out from the timelords, then every node looks at the plots it manages to see if they win. if they win they send out a signal announcing it, then they get recognized with the award. 
Port 8444 is the channel that the nodes talk to eachother on. so a signal over the internet will request to connect to port 8444 on yoru router, then the router will send the message to a computer. You want to log in to your router settings and select the correct local ip address that corresponds to your node, that way your router will send anything from the internet asking for 8444 to your node.

**Reply by tianxinya2021** [reply]
Hi, Thanks for the information. I did not configure any of the router specifically, but assume my node could get signal from the chia net since most of the time the full node works well. I have not got any reward yet since I just started a week ago.

**Reply by dorkmo** [reply]
look at your plot filter to see if you are passing the filter and if you are running up to date with the new blocks.

**Reply by tianxinya2021** [reply]
Yes, blocks are up to date and I saw sometimes plot passing filters. I have firewall open as well.

**Reply by tianxinya2021** [reply]
I see the full node is connected to 
127.0.0.1 | 8444/8444
I would assume everything is fine then.

**Reply by dorkmo** [reply]
that 127.0.0.1 is only the connection between your wallet and your node on that computer. that isnt connected to the internet. you should see a long list of like 50 nodes on your connection list

**Reply by tianxinya2021** [reply]
Yes, there are many connection list on 'full node' page, about 12 of them. 
What I listed above is on 'farm' page of advance options 'Your Full Node Connection'.
